app/src/main/AndroidManifest.xml

```diff
@@ -19,10 +19,10 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.movemeet">
+        <activity android:name=".map.GPSRecordingActivity"></activity>
         <activity
             android:name=".EditProfileActivity"
-            android:windowSoftInputMode="adjustPan" >
-        </activity>
+            android:windowSoftInputMode="adjustPan"></activity>
```

Comment:
You only delete line in the file ?

Reply:
I had so solve a merge conflict in the file and probably deleted the extra characters by accident